app/routes/push.js

```diff
@@ -14,62 +14,61 @@
  * limitations under the License.
  */
 
-import Ember from "ember";
+import Ember from 'ember';
 
-var runLater = function(context, id) {
-  Ember.run.later(context, function() {
-    Ember.$('#push-' + id).remove();
+const runLater = (context, id) => {
+  Ember.run.later(context, () => {
+    Ember.$(`#push-${id}`).remove();
   }, 2000);
 };
 
-var PushRoute = Ember.Route.extend({
-  model: function(){
-    var _this = this;
-    return Ember.$.get('http://localhost:2845/push').then(function(data){
+const PushRoute = Ember.Route.extend({
+  model() {
+    const _this = this;
+    return Ember.$.get('http://localhost:2845/push').then((data) => {
       if (data && data.length > 0) {
-        var message;
-        for (var i=data.length-1; i >=0; i--) {
+        let message;
+        for (let i = data.length - 1; i >= 0; i -= 1) {
           message = data[i];
           if (!_this.lastNotificationTime || _this.lastNotificationTime < message.timestamp) {
-            Ember.$('#push-phone').append('<span class="push-phone-notification" id="push-' + message._id + '">' + message.content + '</span>');
+            Ember.$('#push-phone').append(`<span class="push-phone-notification" id="push-${message._id}">${message.content}</span>`);
             runLater(message._id);
           }
         }
 
         _this.lastNotificationTime = data[data.length - 1].timestamp;
-      }
-      else {
+      } else {
         _this.lastNotificationTime = null;
       }
 
       return data;
     });
   },
-  activate: function() {
+  activate() {
     this.refrshTimer = Ember.run.later(this, this.refreshModel, 3000);
   },
-  deactivate: function() {
+  deactivate() {
     Ember.run.cancel(this.refreshTimer);
   },
-  refreshModel: function() {
+  refreshModel() {
     this.refresh();
     this.refrshTimer = Ember.run.later(this, this.refreshModel, 3000);
   },
   actions: {
-    'refreshNotificationList': function() {
+    refreshNotificationList() {
       Ember.run.cancel(this.refreshTimer);
 
       this.refreshModel();
     },
-    'clearNotificationList': function() {
-      var _this = this;
-      return new Ember.RSVP.Promise(function(resolve) {
-        Ember.$.ajax('http://localhost:2845/push', { type: 'DELETE', statusCode: { 200: function(data){ resolve(data); } } });
-      }).then(function(data){
+    clearNotificationList() {
+      const _this = this;
+      return new Ember.RSVP.Promise(((resolve) => {
```

Comment:
Similar to the above, looks like there's an extra set of parenthesis introduced here/line 67.

Reply:
Fixed. Thanks.